pkg/policy/policies.go

```diff
@@ -84,10 +84,14 @@ func (ps *Policies) Version() uint16 {
 	return ps.version
 }
 
-// ContainerFilterEnabled returns a bitmap of policies that have at least
-// one container filter type enabled.
-func (ps *Policies) ContainerFilterEnabled() uint64 {
-	return atomic.LoadUint64(&ps.containerFiltersEnabled)
```

Comment:
Why not call it atomically?

Reply:
Policy and Policies types are in the process of becoming read-only, so concurrency control will no longer be necessary. The next steps (PRs) will make this clearer.